tools/testbench/common_test.c

```diff
@@ -0,0 +1,264 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
can we get a better name for this file. e.g. utils? This sounds like a common file for the unit test framework

Reply:
Yep, thanks for suggestion! I added to end a commit for renaming. I tried early change but fixing the conflicts seemed a lot of work, so I gave up to save time for other improvements.